src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/modifiableRectangle.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../containers/Application/colors.scss';
+
+$backdropColor: rgba($black, 0.6);
+$borderColor: $white;
+$borderSize: 2px;
+
+.rectangle {
+    position: absolute;
+    background: transparent;
+    border: $borderSize solid $borderColor;
+    cursor: move;
+    overflow: visible;
+    z-index: 1;
+
+    &::after {
+        content: "";
+        position: absolute;
+        top: -$borderSize;
+        left: -$borderSize;
+        bottom: -$borderSize;
+        right: -$borderSize;
+        outline: 20000px solid $backdropColor;
```

Comment:
Although the other solution I proposed does not work: Why can't we set a black-transparent background color on `rectangle`, and revert that change using a filter on the `after` pseudo element?